genie/utils/support.py

```
@@ -77,21 +153,20 @@ def upload_file(content):
+	support_url = frappe.db.get_single_value("Genie Settings", "support_url")
 
-	sid = response.cookies.get("sid")
-	return {
-		"url": f"{settings.support_url}/helpdesk?sid={sid}"
-	}
+	try:
+		return {
+			"url": f"{support_url}/opero/opero/account/login"
+		}
+
+	except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)